incubator/hnc/test/e2e/issues_test.go

```diff
@@ -237,3 +238,80 @@ var _ = Describe("Issues", func() {
 		mustRun("kubectl get rolebinding foo -n", nsChild, "-oyaml")
 	})
 })
+
+var _ = Describe("Issues that require reparing HNC", func() {
+
+	hncRecoverPath := os.Getenv("HNC_REPAIR")
+	const (
+		nsParent = "parent"
+		nsParent2 = "parent-2"
+		nsChild = "child"
+	)
+
+	BeforeEach(func() {// we don't want to destroy the HNC without being able to repair it, so skip this test if recovery path not set
+		if hncRecoverPath == ""{
+			Skip("Environment variable HNC_REPAIR not set. Skipping reocovering HNC.")
+		}
```

Comment:
I know it's only three lines, but maybe eliminate the cut-and-paste?

Reply:
done